From b3e6c20f49361de16dfa74eb8fed176ed913e25b Mon Sep 17 00:00:00 2001
From: mdonthir <reddy.mallikarjun@intel.com>
Date: Fri, 25 Sep 2020 10:40:57 +0800
Subject: [PATCH] UGW_SW-51136: switch_cli mac table static entry write/read
 error

 Irrelevant code is removed which is not applicable for GRX500 platform.

Signed-off-by: Donthireddy Mallikarjuna Reddy <reddy.mallikarjun@intel.com>
---
 drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c b/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
index a04b828345b7..0c0b68d5340f 100644
--- a/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
+++ b/drivers/net/ethernet/lantiq/switch-api/gsw_flow_core.c
@@ -3903,13 +3903,7 @@ GSW_return_t GSW_MAC_TableEntryRead(void *cdev,
 				} else
 					parm->hitstatus = LTQ_FALSE;
 			} else {
-
-				int id = conv_id(tbl_prog.val[0]);
-
-				if (id < 0)
-					parm->nPortId = (u32)tbl_prog.val[0] | 0x80000000;
-				else
-					parm->nPortId = (u32)id;
+				parm->nPortId = (u32)tbl_prog.val[0] | 0x80000000;
 			}
 
 		} else { /*Dynamic Entry*/
